app/views/contexts/calculated_summary_context.py

```diff
@@ -33,6 +33,7 @@ def build_groups_for_section(self, section):
                 self._schema,
                 location,
                 self._language,
+                return_to="final_summary",
```

Comment:
Given that we use `return_to` here with a string value, we should consider changing the url parameters to be the same. So instead of `return_to_summary` and `return_to_final_summary`, we would have `return_to=summary` or `return_to=final_summary` .

Reply:
lol, that was what I originally thought, but was convinced otherwise